gui/default/syncthing/folder/editFolderModalView.html

```diff
@@ -199,24 +199,27 @@
                 <div class="col-md-6 form-group">
                   <label translate>Folder Type</label>
                   &nbsp;<a href="https://docs.syncthing.net/users/foldertypes.html" target="_blank"><span class="fas fa-question-circle"></span>&nbsp;<span translate>Help</span></a>
-                  <select class="form-control" ng-model="currentFolder.type">
+                  <select class="form-control" ng-model="currentFolder.type" ng-change="typeChange()">
                     <option value="sendreceive" translate>Send &amp; Receive</option>
                     <option value="sendonly" translate>Send Only</option>
                     <option value="receiveonly" translate>Receive Only</option>
                   </select>
                   <p ng-if="currentFolder.type == 'sendonly'" translate class="help-block">Files are protected from changes made on other devices, but changes made on this device will be sent to the rest of the cluster.</p>
                   <p ng-if="currentFolder.type == 'receiveonly'" translate class="help-block">Files are synchronized from the cluster, but any changes made locally will not be sent to other devices.</p>
                 </div>
-                <div class="col-md-6 form-group">
+                <div class="col-md-6 form-group" >
                   <label translate>File Pull Order</label>
-                  <select class="form-control" ng-model="currentFolder.order">
+                  <select class="form-control" ng-model="currentFolder.order" ng-show="currentFolder.type != 'sendonly'">
```

Comment:
I think we almost exclusively use ng-if which entirely removes the element from the DOM, while this just hides it. I don't know whether it matters much, but it seems cleaner to remove it entirely than just hide it. You could even put the ng-if on the `option`s, not needing a second select. And I'd also put an `ng-disabled="currentFolder.type == 'sendonly'" on it.